Dockerfile

```diff
@@ -1,79 +1,42 @@
-FROM quay.io/deis/base:v0.3.6
+FROM postgres:11
 
-ENV LANG=en_US.utf8 \
-    PG_MAJOR=9.4 \
-    PG_VERSION=9.4.17-1.pgdg16.04+1 \
-    PGDATA=/var/lib/postgresql/data
+ARG DEBIAN_FRONTEND=noninteractive
+ARG BUILD_DEPS='gcc git libffi-dev libssl-dev python3-dev python3-pip python3-wheel'
 
-# Set this separately from those above since it depends on one of them
-ENV PATH=/usr/lib/postgresql/$PG_MAJOR/bin:$PATH
-
-# Add postgres user and group
-RUN adduser --system \
-    --shell /bin/bash \
-    --disabled-password \
-    --group \
-    postgres
-
-RUN buildDeps='gcc git libffi-dev libssl-dev python3-dev python3-pip python3-wheel' && \
-    localedef -i en_US -c -f UTF-8 -A /etc/locale.alias en_US.UTF-8 && \
-    export DEBIAN_FRONTEND=noninteractive && \
-    apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8 && \
-    echo 'deb http://apt.postgresql.org/pub/repos/apt/ xenial-pgdg main' $PG_MAJOR > /etc/apt/sources.list.d/pgdg.list && \
-    apt-get update && \
+RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-        $buildDeps \
+        $BUILD_DEPS \
         gosu \
         lzop \
-        postgresql-$PG_MAJOR=$PG_VERSION \
-        postgresql-contrib-$PG_MAJOR=$PG_VERSION \
+        libpq-dev \
         pv \
         python3 \
-        postgresql-common \
         util-linux \
         # swift package needs pkg_resources and setuptools
         python3-pkg-resources \
-        python3-setuptools && \
+        python3-setuptools \
+        python3-pip && \
     ln -sf /usr/bin/python3 /usr/bin/python && \
-    ln -sf /usr/bin/pip3 /usr/bin/pip && \
-    mkdir -p /run/postgresql && \
-    chown -R postgres /run/postgresql && \
-    # setuptools from ubuntu archives is too old for googleapis-common-protos
-    pip install --upgrade setuptools && \
+    ln -sf /usr/bin/pip3 /usr/bin/pip
+
+# setuptools from ubuntu archives is too old for googleapis-common-protos
+RUN pip install --upgrade setuptools && \
     pip install --disable-pip-version-check --no-cache-dir \
-        envdir==0.7 \
-        wal-e[aws,azure,google,swift]==v1.0.2 \
-        # pin azure-storage to version wal-e uses (see docker-entrypoint.sh)
-        azure-storage==0.20.0 && \
-    # "upgrade" boto to 2.43.0 + the patch to fix minio connections
-    pip install --disable-pip-version-check --no-cache-dir --upgrade git+https://github.com/teamhephy/boto@88c980e56d1053892eb940d43a15a68af4ebb5e6 && \
-    # cleanup
-    apt-get purge -y --auto-remove $buildDeps && \
+        envdir==1.0.1 \
+        wal-e[aws,azure,google,swift]==1.1.0 \
+        gcloud==0.18.3 \
+        oauth2client==4.1.3 \
+        azure-storage==0.20.0
+
+# cleanup
+RUN apt-get purge -y --auto-remove $buildDeps && \
```

Comment:
This should be `RUN apt-get purge -y --auto-remove $BUILD_DEPS...`? :question:

Reply:
Yes, thanks; I'll fix it.

Reply:
Looks good!